CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.23.0] - 2021-12-28
+
 ### Added
 - Add support for tracing with OpenTelemetry. This adds a new function to the authenticator, `AuthenticateWithContext`. The existing funtion, `Authenticate()` is deprecated and will be removed in a future upddate. [cyberark/conjur-authn-k8s-client#423](https://github.com/cyberark/conjur-authn-k8s-client/pull/423)
```

Comment:
Would you mind adding an `Added` entry here that probably should have been added earlier:
```
- Add support for configuration via Pod Annotations. [[cyberark/conjur-authn-k8s-client#407](https://github.com/cyberark/conjur-authn-k8s-client/pull/407)
```

Also, would you mind changing the first entry in the `Changed` list for this version to say that we're going from Golang `v1.15` to `v1.17` (instead of `v1.16`), and add a link to a second PR. That is, change this line:
```
- The project Golang version is updated from the end-of-life v1.15 to version v1.16. [cyberark/conjur-authn-k8s-client#416](https://github.com/cyberark/conjur-authn-k8s-client/pull/416)
```
To this:
```
- The project Golang version is updated from the end-of-life v1.15 to version v1.17.
  [cyberark/conjur-authn-k8s-client#416](https://github.com/cyberark/conjur-authn-k8s-client/pull/416)
  [cyberark/conjur-authn-k8s-client#424](https://github.com/cyberark/conjur-authn-k8s-client/pull/424)
```